plugins/stack-overflow/package.json

```diff
@@ -54,6 +54,7 @@
     "@testing-library/jest-dom": "^6.0.0",
     "@types/react": "^16.13.1 || ^17.0.0",
     "cross-fetch": "^4.0.0",
+    "he": "^1.2.0",
```

Comment:
Oof, 124 kB for this! That's heavy for a one-time use. Just noting.

You need to run `yarn install` and commit the `yarn.lock` changes along with the PR.

Reply:
Noted, I'll see if I can come up with a better solution!